src/model.rs

```diff
@@ -159,7 +159,7 @@ impl<B: Backend<FloatElem = f32>> Model<B> {
             // dbg!(&difficulty);
             // dbg!()
         }
-        (stability, difficulty)
+        (weight_clipper(stability), weight_clipper(difficulty))
```

Comment:
The stability and difficulty are not the parameters. I mean `self.w`.

Reply:
`pub w: Param<Tensor<B, 1>>` is the parameter.

Reply:
Sorry. like that?

Reply:
hold on nvm